src/diffusers/models/attention_processor.py

```diff
@@ -72,7 +73,8 @@ def __init__(
         self.upcast_attention = upcast_attention
         self.upcast_softmax = upcast_softmax
 
-        self.scale = dim_head**-0.5 if scale_qk else 1.0
+        self.scale_qk = scale_qk
+        self.scale = dim_head**-0.5 if self.scale_qk else 1.0
```

Comment:
Why this change? (Just curious)

Reply:
Cannot use the `scale_qk` parameter in the latter methods otherwise. This parameter is for now needed to set the default attention processor to `AttnProcessor2_0`. 

I followed how we are setting `processor` in the init for that.